src/wlserver.cpp

```diff
@@ -1042,22 +1043,51 @@ bool wlserver_surface_is_async( struct wlr_surface *surf )
 	return get_wl_surface_info( surf )->presentation_hint != 0;
 }
 
+/* Handle the orientation of the touch inputs */
+void get_effective_touchscreen_orientation(double *x, double *y )
```

Comment:
Maybe `apply_touchscreen_orientation` would be a better name?

Reply:
Done